### ISSUE TITLE
todo delete auto-completing to todo delete -f (and then automatically runs)

### ISSUE THREAD

**Issue by zebak12** [pimutils/todoman#143]
When writing ```todo del``` and then press tab, this appears:
```
todo del......Error: no such option: -f
Error: no such option: -f
Error: no such option: -f
```

I am running zsh
PS: I have enabled those dots (```......```)  while loading auto-completion

**Comment by untitaker** [comment]
I don't understand, `todoman` doesn't offer tab-completion, or does it @hobarrera?

**Comment by hobarrera** [comment]
Nope, nor todoman, nor the AUR package. What distribution is this, @zebak12 ? Do you know where the auto-completion is coming from?

**Comment by zebak12** [comment]
I am sorry for the late reply, I was out of station for a while!
>Nope, nor todoman, nor the AUR package. What distribution is this, @zebak12 ? Do you know where the auto-completion is coming from?

Regarding the distribution, I am using zsh on Ubuntu 16.04.1 LTS 64-bit

**Comment by untitaker** [comment]
This appears to be an issue with zsh (or an extension), not todoman, though I have no idea what exactly is happening. I can't tell you anything else than to avoid hitting <key>tab</key>.

**Comment by hobarrera** [comment]
FWIW, the Arch package for zsh, has autocompletion for some other app named `todo` (from todo.sh, apparently, and not in the base repos though).

**Comment by untitaker** [comment]
I was actually wondering if todoman's binary name might not be a bit too generic.

**Comment by hobarrera** [comment]
On arch, no package provides `/usr/bin/t` nor `/usr/bin/td`. I like the binary name, TBH, since it's clear, short and descriptive.

**Comment by untitaker** [comment]
`t` is already taken by a Twitter client in AUR. I'd rather go for `todoman` and let people alias it.

**Comment by zebak12** [comment]
> FWIW, the Arch package for zsh, has autocompletion for some other app named todo (from todo.sh, apparently, and not in the base repos though).

Oh! Thanks @hobarrera 
>`t` is already taken by a Twitter client in AUR. I'd rather go for todoman and let people alias it.

Yeah, that would be a great!

**Comment by hobarrera** [comment]
Okay to close this?

**Comment by untitaker** [comment]
Yes I think so

On 9 March 2017 03:26:02 CET, Hugo Osvaldo Barrera <notifications@github.com> wrote:
>Okay to close this?
>
>-- 
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/pimutils/todoman/issues/143#issuecomment-285235531

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by untitaker** [comment]
However, I do think we should rename the binary such that it doesn't conflict with todo.txt

**Comment by RJ722** [comment]
Hello everyone,

I am not sure if this is the same thing, but I can reproduce a somewhat similar version of this issue:

```
$ todo <TAB>todo Usage: todo [OPTIONS] COMMAND [ARGS]...
Try "todo --help" for help.

Error: no such option: -f
Usage: todo [OPTIONS] COMMAND [ARGS]...
Try "todo --help" for help.

Error: no such option: -f
Usage: todo [OPTIONS] COMMAND [ARGS]...
Try "todo --help" for help.

Error: no such option: -f
```

My machine runs macOS - zsh on iTerm2. I have added the `_todo`file in `/usr/local/share/zsh/site-functions`.

Also, one it's done printing the error message, it waits for input and I can write the command for todoman and it will run fine, for example:

```
$ todo Usage: todo [OPTIONS] COMMAND [ARGS]...
Try "todo --help" for help.

Error: no such option: -f
Usage: todo [OPTIONS] COMMAND [ARGS]...
Try "todo --help" for help.

Error: no such option: -f
Usage: todo [OPTIONS] COMMAND [ARGS]...
Try "todo --help" for help.

Error: no such option: -f
list
33  [ ]       in a day     mock_todo_item
```

I'm happy to provide any further information!  I've been trying hard to not hit TAB, but that is super hard and it is keeps gets frustrating 😅, especially because (as you can see above) the error is thrown thrice.

P.S. Thanks for the wonderful tool! :-)

**Comment by rtur** [comment]
@RJ722 I had the same issue and at first assumed there is a conflicting completion provider also. It turned out it was a somehow corrupted completion database. For me it was enough to delete the ~/.zcompdump file and everything worked again.

**Comment by RJ722** [comment]
It's working again! \o/

Thank you so much @rtur! :-)

 @untitaker @WhyNotHugo do you think it's worth documenting this?
